ansible/vault_nomad_integration.yml

```diff
@@ -0,0 +1,57 @@
+- name: Start vault's nomad secrets backend
+  shell:
+    cmd: vault secrets enable nomad
+  environment:
+    VAULT_ADDR: http://127.0.0.1:8200
+    VAULT_TOKEN: master
+
+- name: fetch bootstrap token nomad
+  shell:
+    cmd: vault kv get -field=secret-id secret/nomad-bootstrap-token
+  register: bootstrap
+  environment:
+    VAULT_TOKEN: master
+    VAULT_ADDR: http://127.0.0.1:8200
+
+- name: Create management token for vault
+  shell:
+    cmd: nomad acl token create -name="Vault Management Token" -type="management" > management.token
+  environment:
+    NOMAD_TOKEN: "{{ bootstrap.stdout }}"
+
+- name: register accessor ID
+  shell:
+    cmd: awk '/Accessor/ {print $4}' management.token
+  register: accessor_id
+
+- name: register secret ID
+  shell:
+    cmd: awk '/Secret/ {print $4}' management.token
+  register: secret_id
+
+- name: add nomad management token to vault
+  shell:
+    cmd: vault kv put secret/nomad-management-token accessor-id="{{ accessor_id.stdout }}" secret-id="{{ secret_id.stdout }}"
+  environment:
+    VAULT_ADDR: http://127.0.0.1:8200
+    VAULT_TOKEN: master
```

Comment:
WDYT to put VAULT_TOKEN value `master` to default_variables?

Reply:
I've been thinking of the `default_variables`-file as a sort of "These are variables that can be used to alter the system, and there are their default values". The vault token would not be changed on a frequent basis, and not by the user (only those developing the box itself), therefore it doesn't feel right to me putting it in `default_variables`. 

What DOES feel right is maybe having environment variables like NOMAD_TOKEN, VAULT_TOKEN, VAULT_ADDR etc. set when the box starts. Didn't you try and figure something like that out when working out how to pass variables to ansible, and realise there was no easy way?

Reply:
There is opportunity to pass variables via extra vars. But I think in ur case u can setup env variables, just to append veriable in the .bashrc 
[Example](https://github.com/fredrikhgrelland/vagrant-hashistack/blob/master/ansible/install.yml#L125-L131)

Reply:
Right, that's a quick and easy way. I'll get on that

Reply:
Update: this should be changed, but I'll bundle that with another PR instead